Doc/library/multiprocessing.rst

```diff
@@ -2918,12 +2918,12 @@ Beware of replacing :data:`sys.stdin` with a "file like object"
                self._cache = []
            return self._cache
 
-    For more information, see :issue:`5155`, :issue:`5313` and :issue:`5331`
+    For more information, see :issue:`5155`, :issue:`5313` and :issue:`5331`.
```

Comment:
You could update these to the GitHub numbers and the `:gh:` role.

Reply:
Sure, I have updated line 2921. Do the changes look okay? Thanks!